config/routes.rb

```diff
@@ -770,6 +770,14 @@
 
     resources :organisation_settings, path: '/organisation-settings', only: :index
 
+    resources :organisation_permissions_setup, only: %i[index edit update], path: 'organisation-permissions/setup' do
```

Comment:
Maybe we could switch this round and use a `setup` scope, and nest the resource in there, so the URL would be /setup/organisation-permissions